react/AccordionItem/AccordionItem.demo.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { AccordionItem, Text, PageBlock, Section } from 'seek-style-guide/react';
+
+class AccordionContainer extends Component {
+  static propTypes = {
+    component: PropTypes.func.isRequired,
+    componentProps: PropTypes.object.isRequired
+  };
+
+  render() {
+    const { component: DemoComponent, componentProps } = this.props;
+    return (
+      <PageBlock>
+        <Section>
+          <DemoComponent
+            {...componentProps}
+          />
+        </Section>
+      </PageBlock>
+    );
+  }
+}
+
+export default {
+  route: '/accordionitem',
+  title: 'Accordion Item',
+  category: 'Layout',
+  component: AccordionItem,
+  container: AccordionContainer,
+  block: true,
+  initialProps: {
+    title: <Text>Living Style Guide</Text>,
```

Comment:
This should be vertically centred rather than on grid.
```suggestion
    title: <Text baseline={false} raw>Living Style Guide</Text>,
```